rosplane/CMakeLists.txt

```diff
@@ -33,59 +33,75 @@ find_package(Eigen3 3.3 REQUIRED NO_MODULE)
 find_package(rosflight_msgs REQUIRED)
 pkg_check_modules(YAML_CPP REQUIRED yaml-cpp)
 
-ament_export_dependencies(rclcpp rclpy)
-ament_export_include_directories(include)
```

Comment:
Why do we not need these? I am curious, were they ever necessary? It seems like they were moved to each executable.

Reply:
I don't believe so. Those two lines are used if you ever want to use ROSplane as a dependency for other packages, but you can also create and install a library like I did which is more explicit and limited in scope, I think. I'll also be the first to admit I don't understand CMake very well. At the very least, everything still works without those two lines and they weren't in the examples I looked at.